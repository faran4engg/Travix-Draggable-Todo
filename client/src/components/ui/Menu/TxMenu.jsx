import React from "react";
import { Segment } from "semantic-ui-react";
import PropTypes from "prop-types";
import styled from "styled-components";
import TxHeading from "../Heading/TxHeading";

const StyledSegment = styled(Segment)`
  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23) !important;
  border-radius: 0 !important;
`;

const TxMenu = props => {
  return (
    <React.Fragment>
      <StyledSegment inverted textAlign="center">
        <TxHeading content="Travix" path={props.path} />
      </StyledSegment>
    </React.Fragment>
  );
};

TxMenu.propTypes = {
  path: PropTypes.string
};

export default TxMenu;
