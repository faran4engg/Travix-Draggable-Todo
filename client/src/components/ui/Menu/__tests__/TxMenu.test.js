import React from "react";
import { shallow } from "enzyme";
import TxMenu from "../TxMenu";

describe("TxMenu", () => {
  it("should render correctly ", () => {
    const component = shallow(<TxMenu />);
    expect(component).toMatchSnapshot();
  });
});
