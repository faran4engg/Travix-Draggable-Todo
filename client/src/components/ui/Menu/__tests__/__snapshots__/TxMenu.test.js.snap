// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TxMenu should render correctly  1`] = `ShallowWrapper {}`;
